.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -124,8 +127,8 @@ body:
     id: dev-release
     attributes:
       label: I have tested this issue on a development release
-      placeholder: discord.js@xx.x.x-dev.1530234593.d23280c
+      placeholder: d23280c
```

I believe this should say the entire version number to make it easier to install the specified version when testing
```suggestion
      placeholder: xx.x.x-dev.1530234593.d23280c
```

---

no

---

Why 😐

---

this inhibits embedding of the commit preview + short link, as mentioned in the thread. Parallel discussions in multiple places are not efficient or pleasant.